playground/TestShop/AppHost/Program.cs

```diff
@@ -17,9 +17,11 @@
                            .WithReference(basketCache)
                            .WithReference(messaging);
 
+var isHttps = string.Equals(builder.Configuration["DOTNET_LAUNCH_PROFILE"], "https", StringComparison.OrdinalIgnoreCase);
```

Comment:
Is there not a constant buried away somewhere or an enum so this doesn't feel like magic strings?